### ISSUE TITLE
Documentation site not indexed by search engines

### ISSUE THREAD

**Issue by colinodell** [kubernetes-sigs/cli-experimental#382]
**TL;DR:** Content from https://kubectl.docs.kubernetes.io/ cannot be found using popular search engines.

I expected to be able to find appropriate documentation pages like https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/ when searching for things like `kustomization reference` on Google. However, I'm unable to find results for this site when I run searches like:

- https://www.google.com/search?q=kustomization+reference
- https://www.google.com/search?q=site%3Akubectl.docs.kubernetes.io
- https://www.bing.com/search?q=site%3Akubectl.docs.kubernetes.io (XML files only; no HTML results)

I believe this is happening because of this meta tag in the HTML:

```html
<META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
```

It looks like this is only supposed to appear when `HUGO_ENV` != `production`:

https://github.com/kubernetes-sigs/cli-experimental/blob/3445a582efe9308f4f81bd6da23ad092e9e89d92/site/themes/docsy/layouts/partials/head.html#L4-L8

So maybe the build environment is failing to set this env var correctly?

**Comment by colinodell** [comment]
This issue is still present and worth fixing

/remove-lifecycle stale

**Comment by colinodell** [comment]
This is still very much a problem

/remove-lifecycle stale

**Comment by brianpursley** [comment]
This appears to be related to the [Docsy](https://github.com/google/docsy) theme.  It looks like it was later changed to use `hugo.IsProduction`
https://github.com/google/docsy/blame/16335ca146516a94a6ca16cc7a755d18ac02940d/layouts/_partials/head.html#L9-L13

Should the Docsy theme be updated in this project, or has it been customized?  Or maybe we should just change to use `hugo.IsProduction`.  I don't know enough about hugo to know if `HUGO_ENV` is still a valid way to check the environment.  Maybe it was deprecated or something.

I guess we could try adding something like this to the generated output that prints what it *thinks* the environment is, to help us troubleshoot the problem:
```
<!-- debug: env={{ printf "%q" hugo.Environment }}, IsProd={{ hugo.IsProduction }}, HUGO_ENV={{ printf "%q" (getenv "HUGO_ENV") }}, HUGO_ENVIRONMENT={{ printf "%q" (getenv "HUGO_ENVIRONMENT") }} -->
```

Thoughts?
